mlrun/model_monitoring/batch.py

```diff
@@ -600,6 +601,13 @@ def _initialize_v3io_configurations(self):
             container=self.tsdb_container,
             token=self.v3io_access_key,
         )
+        logger.info("Creating table in TSDB", table=self.tsdb_path)
+        self.frames.create(
```

Comment:
where it's created from on set_tracking?

Reply:
It's through the monitoring deployment in the `EventStreamProcessor` class:
https://github.com/mlrun/mlrun/blob/51a0c93b79ce40fe931364abb946993b91fc6611/mlrun/model_monitoring/stream_processing.py#L337-L357